src/insns/pcrel_debug_warning.adoc

```diff
@@ -1,2 +1,4 @@
 NOTE: The instructions on this page are either PC relative or may update the
-<<pcc>>. Therefore an implementation may make them illegal in debug mode.
+<<pcc>>. Therefore an implementation may make them illegal in debug mode. If
+not then the value of the <<pcc>> in debug mode is UNSPECIFIED by this
```

Comment:
```suggestion
they are supported then the value of the <<pcc>> in debug mode is UNSPECIFIED by this
```

Reply:
I will need to read the debug spec to ensure that whatever we say here matches the debug spec, but maybe we say PCC has almighty permissions/bounds and whatever address mandated by the debug spec?